src/odemis/odemisd/test/modelgen_test.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on 16 Dec 2020
+
+@author: Kornee Kleijwegt
+
+Copyright © 2019-2021 Kornee Kleijwegt, Delmic
+
+This file is part of Delmic Acquisition Software.
+
+Delmic Acquisition Software is free software: you can redistribute it and/or modify it under the terms of the GNU
+General Public License as published by the Free Software Foundation, either version 2 of the License, or (at your
+option) any later version.
+
+Delmic Acquisition Software is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+more details.
+
+You should have received a copy of the GNU General Public License along with Delmic Acquisition Software. If not, see
+http://www.gnu.org/licenses/.
+"""
+import unittest
+import yaml
+
+from odemis.odemisd.modelgen import SafeLoader, ParseError
+
+
+class SafeLoader_extensions_test(unittest.TestCase):
```

Comment:
This mix of cases looks strange. I would go for `SafeLoaderExtensionsTest`, this is also the convention in the rest of odemis.